packages/integration_test/lib/integration_test_driver.dart

```diff
@@ -70,6 +70,7 @@ Future<void> writeResponseData(
 Future<void> integrationDriver({
   Duration timeout = const Duration(minutes: 20),
   ResponseDataCallback? responseDataCallback = writeResponseData,
+  bool writeResponseOnFailure = false,
```

Comment:
This parameter needs to be documented above.

Reply:
added